app/models/miq_user_role.rb

```diff
@@ -37,8 +37,8 @@ def feature_identifiers
   def allows?(identifier:)
     if feature_identifiers.include?(identifier)
       true
-    elsif parent = MiqProductFeature.parent_for_feature(identifier)
-      allows?(:identifier => parent.identifier)
+    elsif (parent = MiqProductFeature.feature_parent(identifier))
+      allows?(:identifier => parent)
```

Comment:
`parent_for_feature` definitely can be removed (I originally thought it gave you the identifier and `feature_parent` gave you the object, but it looks like they do the same thing?) but this object vs. identifier problem seems to remain. It doesn't look like you should be able to pass an entire MiqProductFeature object into this method (only the identifier). Yes/no?